src/wallet.rs

```diff
@@ -783,15 +783,31 @@ fn request_and_confirm_airdrop(
         };
         match status {
             RpcSignatureStatus::AccountInUse => {
-                last_id = get_last_id(rpc_client)?;
+                // Fetch a new last_id, to prevent the retry from getting rejected as a
```

Comment:
Why are you implementing a DuplicateSignature handler in the AccountInUse handler?

Reply:
I'm not.  When `AccountInUse` happens, we need to fetch a new last_id to prevent `DuplicateSignature` on the next retry

Reply:
got it, thanks!